PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+#### What does this PR do?
+#### Where should the reviewer start?
+#### How should this be manually tested?
+#### Any background context you want to provide?
+#### Screenshots (if appropriate)
+#### Questions:
+- Have you connected this PR to the issue it fixes?
+- Does the documentation need an update?
```

Comment:
Have you added unit or functional tests for this patch?

Reply:
Another good one. Added: "Have you added unit or functional tests for this PR?"